src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,296 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static Calculator calculator;
+    private static final double DELTA = 0.0001;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new Calculator();
+    }
+
+    //Addition
+    @Test
+    void additionWithPositivesOperands_Ok() {
```

Comment:
```suggestion
    void calculate_additionWithPositiveOperands_ok() {
```
same for other methods
we should indicate which method we are testing